### PR TITLE
add stackdriver logging libraries

### PR DESCRIPTION
**Pull request by elharo**

### CODE REVIEW

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml [hunk 1]
```diff
@@ -308,7 +308,7 @@
         <mavenCoordinates
               artifactId="google-cloud-datastore"
               groupId="com.google.cloud"
-              version="1.0.1" />
+              version="LATEST" />
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: Looking at [google-cloud-datastore](https://repo1.maven.org/maven2/com/google/cloud/google-cloud-datastore/1.4.0/google-cloud-datastore-1.4.0.pom), it also requires:
  - `com.google.cloud.datastore:datastore-v1-protos:1.3.0`
  - `com.google.cloud.datastore:datastore-v1-proto-client:1.3.0`
  - `com.google.api.grpc:grpc-google-common-protos:0.1.15`

File: plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml [hunk 2]
```diff
@@ -325,7 +325,24 @@
...
+    </library>
+
+    <library
+          id="cloudlogging"
+          group="clientapis"
+          name="%stackdriverlogging.name"
+          tooltip="%stackdriverlogging.tooltip"
+          javaVersion="1.8"
+          siteUri="https://cloud.google.com/translate/docs/reference/libraries" >
+      <libraryDependency id="googlecloudcore" />
```
Comment (briandealwis) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: I don't think this one is so easy: [`com.google.cloud:google-cloud-logging:1.4.0`](https://repo1.maven.org/maven2/com/google/cloud/google-cloud-logging/1.4.0/google-cloud-logging-1.4.0.pom) shows that it requires:
   + `io.netty:netty-tcnative-boringssl-static:jar:2.0.3.Final`
   + `com.google.cloud:google-cloud-core-grpc:jar:1.4.0`
   + `com.google.api:gax-grpc:jar:0.22.0`
   + `com.google.api.grpc:proto-google-cloud-logging-v2:jar:0.1.15`
   + `io.grpc:grpc-netty:jar:1.4.0`
   + `io.grpc:grpc-stub:jar:1.4.0`
```

Comment (elharo) [inline plugins/com.google.cloud.tools.eclipse.appengine.libraries/plugin.xml]: I think the latter group is going to be a req for any library that uses the GRPC transport.

**Comment by patflynn** [comment]
Let's stop adding libraries to this feature.  I'd rather wind down this use-case and focus on the new API wizard flows. 

Feel free to ping me off of github to discuss.

**Comment by patflynn** [comment]
@elharo 
let's close out this PR. Should not be merged.

**Pull request closed without merging**